kupfer/plugin/websearch.py

```diff
@@ -18,7 +18,7 @@
 from kupfer.objects import TextLeaf
 from kupfer import utils, config
 
-from kupfer.plugin import firefox_support
+from kupfer.plugin import firefox
```

Comment:
I'm pretty :-1: on supporting more opaque formats. It's hard to say if it will continue working, say 2 years from now when current kupfer is in debian stable or something.